### PR TITLE
Updated docs

### PR DESCRIPTION
**Pull request by StevenJSCF**
**This pull request aims to enhance the usability and deployment process of OpenVoice by introducing a Dockerfile. Additionally, updates have been made to the USAGE.md document to provide clearer instructions for installing both OpenVoice V1 and V2.**

Fixes issues: 
- #174 
- #154

### CODE REVIEW

**Comment by Zengyi-Qin** [comment]
Thank you! We are reviewing the PR

**Comment by Zengyi-Qin** [comment]
Let us first list your contribution here: https://github.com/myshell-ai/OpenVoice/blob/main/docs/USAGE.md#install-on-other-platforms

**Review by Isiah-n-paul85 — APPROVED** [review]

**Comment by Perdolique** [comment]
`F` in `Dockerfile` filename should be lowercase. Otherwise, the docker command will not recognize it.

```
ERROR: failed to solve: failed to read dockerfile: open Dockerfile: no such file or directory
```

Also, I've got the next output with this `Dockerfile`:

```
=> ERROR [ 2/16] RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y     sudo     python3.9     python3-distutils     python3-pip     ffmpeg     git                6.7s
------
 > [ 2/16] RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y     sudo     python3.9     python3-distutils     python3-pip     ffmpeg     git:
0.612 Get:1 http://ports.ubuntu.com/ubuntu-ports noble InRelease [256 kB]
1.120 Get:2 http://ports.ubuntu.com/ubuntu-ports noble-updates InRelease [89.7 kB]
1.305 Get:3 http://ports.ubuntu.com/ubuntu-ports noble-backports InRelease [90.8 kB]
1.487 Get:4 http://ports.ubuntu.com/ubuntu-ports noble-security InRelease [89.7 kB]
1.671 Get:5 http://ports.ubuntu.com/ubuntu-ports noble/multiverse arm64 Packages [274 kB]
1.829 Get:6 http://ports.ubuntu.com/ubuntu-ports noble/universe arm64 Packages [19.0 MB]
5.151 Get:7 http://ports.ubuntu.com/ubuntu-ports noble/main arm64 Packages [1776 kB]
5.309 Get:8 http://ports.ubuntu.com/ubuntu-ports noble/restricted arm64 Packages [113 kB]
5.321 Get:9 http://ports.ubuntu.com/ubuntu-ports noble-updates/universe arm64 Packages [14.2 kB]
5.321 Get:10 http://ports.ubuntu.com/ubuntu-ports noble-updates/main arm64 Packages [24.0 kB]
5.321 Get:11 http://ports.ubuntu.com/ubuntu-ports noble-security/universe arm64 Packages [8901 B]
5.544 Get:12 http://ports.ubuntu.com/ubuntu-ports noble-security/main arm64 Packages [23.7 kB]
5.790 Fetched 21.8 MB in 6s (3953 kB/s)
5.790 Reading package lists...
6.237 Reading package lists...
6.544 Building dependency tree...
6.606 Reading state information...
6.633 Package python3-distutils is not available, but is referred to by another package.
6.633 This may mean that the package is missing, has been obsoleted, or
6.633 is only available from another source
6.633
6.634 E: Unable to locate package python3.9
6.634 E: Couldn't find any package by glob 'python3.9'
6.634 E: Couldn't find any package by regex 'python3.9'
6.634 E: Package 'python3-distutils' has no installation candidate
------
Dockerfile:5
--------------------
   4 |     # Update the system and install necessary dependencies
   5 | >>> RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y \
   6 | >>>     sudo \
   7 | >>>     python3.9 \
   8 | >>>     python3-distutils \
   9 | >>>     python3-pip \
  10 | >>>     ffmpeg \
  11 | >>>     git
  12 |
--------------------
ERROR: failed to solve: process "/bin/sh -c apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y     sudo     python3.9     python3-distutils     python3-pip     ffmpeg     git" did not complete successfully: exit code: 100
```

File: docs/DF_USAGE.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+This Dockerfile provides a convenient way to set up an environment for running OpenVoice, a project by MyShell AI, on an Ubuntu base image. OpenVoice is a tool for voice manipulation and conversion.
+
+## Prerequisites
+
+Before you can use this Docker image, you need to have Docker installed on your system.
+
+### Installing Docker
+
+Follow the instructions on the [official Docker website](https://docs.docker.com/get-docker/) to install Docker for your operating system.
```
Comment (Isiah-n-paul85) [inline docs/DF_USAGE.md:11]: docker build -t myshell-openvoice .

File: docs/DF_USAGE.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+## Running OpenVoice
+To run OpenVoice with the Docker image, you can use the following example command:
+
+```
+bash
+docker run -it -p 7860:7860 myshell-openvoice
+```
+
+Now you have a Docker container ready to run the OpenVoice application. Access the OpenVoice application at http://localhost:7860 in your web browser.
```
Comment (Isiah-n-paul85) [inline docs/DF_USAGE.md:29]: docker run -it -p 7860:7860 myshell-openvoice

Comment (Isiah-n-paul85) [inline docs/DF_USAGE.md:29]: docker run -it -p 7860:7860 myshell-openvoice

File: docs/DF_USAGE.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+
+## References
+- [OpenVoice MyShell GitHub Repository](https://github.com/myshell-ai/OpenVoice)
```
Comment (Isiah-n-paul85) [inline docs/DF_USAGE.md:33]: docker run -it -p 7860:7860 myshell-openvoice

File: docs/USAGE.md [hunk 4]
```diff
@@ -33,18 +38,35 @@ For users who want to quickly try OpenVoice and do not require high quality or s
...
-```
+### Common Installation Steps (V1 and V2)
 
 No matter if you are using V1 or V2, the above installation is the same.
 
+**1. Clone the Repository:**
+    ```
+    bash
+    git clone git@github.com:myshell-ai/OpenVoice.git
+    cd OpenVoice
```
Comment (Isiah-n-paul85) [inline docs/USAGE.md:51]: conda create -n openvoice python=3.9 
 conda activate openvoice 
 git clone git@github.com:myshell-ai/OpenVoice.git 
 cd OpenVoice 
 pip install -e . 
 ```

File: docs/USAGE.md [hunk 5]
```diff
@@ -33,18 +38,35 @@ For users who want to quickly try OpenVoice and do not require high quality or s
...
-pip install -e .
-```
+### Common Installation Steps (V1 and V2)
 
 No matter if you are using V1 or V2, the above installation is the same.
 
+**1. Clone the Repository:**
+    ```
+    bash
+    git clone git@github.com:myshell-ai/OpenVoice.git
```
Comment (Isiah-n-paul85) [inline docs/USAGE.md:50]: pip install -e .

**Review by Isiah-n-paul85 — APPROVED** [review]

**Pull request closed without merging**